battleship-client/src/main/java/com/epam/solid/nie/client/Main.java

```diff
@@ -2,7 +2,8 @@
 
 public class Main {
     public static void main(String[] args) {
-        ShipClient shipServer = new ShipSocketClient();
+        System.out.println("server ip is: " + (args.length > 0 ? args[0] : ""));
+        ShipClient shipServer = new ShipSocketClient(args[0]);
```

Comment:
nazwa zmiennej shipClient zamiast shipServer

Reply:
changed in last commit